packages/domain-search/src/components/search-bar/filter.tsx

```diff
@@ -0,0 +1,124 @@
+import { Dropdown } from '@wordpress/components';
+import { useCallback, useMemo, useState } from 'react';
+import { useDomainSearch } from '../../page/context';
+import { FilterState } from '../../page/types';
+import { DomainSearchControls } from '../../ui';
+import type { TokenItem } from '@wordpress/components/build-types/form-token-field/types';
+
+const emptyFilter: FilterState = {
+	exactSldMatchesOnly: false,
```

Comment:
Can we rename this variable and remove the `sld` part? Or at least rename to something more obvious? I'm afraid future maintainers might not know what this means.

Reply:
Yeah, I think `exactMatchesOnly` is fine and clear.